### PR TITLE
Update server instructions

### PR DESCRIPTION
**Pull request by pvanderknyff**
Tested this a bit ago. Doesn't require messing about with tsm options.

### CODE REVIEW

**Comment by joe-langford** [comment]
On Windows, can a "server node" have multiple drives, say c: and d:? Reason I ask is that for dev plugins, anyway, my understanding (I got tripped up by it) is that the connector directory has to be on the same drive as the server install. Not sure this would trip anyone up here, but thought you might want to double-check wording of option 2.

**Comment by pvanderknyff** [comment]
> On Windows, can a "server node" have multiple drives, say c: and d:? Reason I ask is that for dev plugins, anyway, my understanding (I got tripped up by it) is that the connector directory has to be on the same drive as the server install. Not sure this would trip anyone up here, but thought you might want to double-check wording of option 2.

I have very little knowledge of how server nodes work, I thought it was a 1-to-1 node-machine relationship. I should probably say "machine" and not "node" in the second option, but in reality someone should go through this process and double check how connector plugins work on nodes.

**Comment by rosswbrown** [comment]
Does this mean they end up in the same directory as partner native connectors? Or are those in a different location after install?

**Comment by pvanderknyff** [comment]
> Does this mean they end up in the same directory as partner native connectors? Or are those in a different location after install?

The system plugins are somewhere else. This is the "repository" for vizqlserver, it's the same folder structure as "My Tableau Repository".

File: docs/run-taco.md [hunk 1]
```diff
@@ -9,9 +9,16 @@ Starting in 2019.4 Beta 1, you can load packaged connectors (otherwise known as
 1. Launch Tableau Desktop.
 
 ## Run a packaged connector in Tableau Server
-1. Create a directory for Tableau connectors. For example:   
+### Option 1
```
Comment (rosswbrown) [inline docs/run-taco.md:12]: Option 1 I think is likely the preferred path.  Option 2 is basically the developer and non-packaged, but seems good to document both in the SDK at least.

**Pull request merged**